xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/streaming/OkCoinBaseStreamingService.java

```diff
@@ -68,4 +66,16 @@ public void send(String msg) {
   public READYSTATE getWebSocketStatus() {
     return READYSTATE.OPEN;
   }
+
+  public WebSocketBase getSocketBase() {
```

Comment:
I would not expose internals of the class. I would consider making this method protected so it can be used by descendants and that's it! Or I would change filed 'socketBase' from private to protected